### ISSUE TITLE
Invalid prop `field` of type `array` supplied to `Field`

### ISSUE THREAD

**Issue by angelikatyborska** [react-tools/react-form#254]
After updating to v3.0.0 I started getting this warning for all fields that I use for with items:
```
Warning: Failed prop type: Invalid prop `field` of type `array` supplied to `Field`, expected `string`.
```

The code that causes this warning is:
```
<Text field={[index, 'value']} />
```

I noticed that you added `propTypes` to the `Field` component which states that the `field` prop needs to be a string: https://github.com/react-tools/react-form/blob/a30200999317e7353be04bf8a745626045e63236/src/components/Field.js#L166 However, your docs still list in many many places examples of the `field` prop that is an array, eg. https://github.com/react-tools/react-form/blob/a30200999317e7353be04bf8a745626045e63236/demo/src/FieldSyntax.js#L26

**Comment by joepuzzo** [comment]
Yup i will get a fix out today. You are totally right

**Comment by joepuzzo** [comment]
This is now fixed in V3.0.3
